app/policies/media_reaction_policy.rb

```diff
@@ -0,0 +1,10 @@
+class MediaReactionPolicy < ApplicationPolicy
+  def create?
+    record.user == user
```

Comment:
Probably needs to have a separate `create?` from `destroy?` so that admins can delete reported reviews